backend/validators/UserValidation.js

```diff
@@ -21,16 +21,16 @@ validators.validateEmail = (email) => {
 };
 
 validators.validatePassword = (pass) => {
-  if (pass.length < 8) {
-    let err = new Error("Password should be atleast 8 characters long");
+  if (pass?.length < 8) {
```

Comment:
what you did is correct but it doesn't work while working with ReactJs or Postman